lib/rmt/cli/repos.rb

```diff
@@ -1,13 +1,17 @@
 class RMT::CLI::Repos < RMT::CLI::Base
 
-  default_task :list
+  include ::RMT::CLI::RepoPrintable
+
+  desc 'custom', 'Custom Repositories'
```

Comment:
all other descriptions follow the pattern "verb object bla", see yourself with `rmt-cli repos`. So until we come up with something better, I would change it to "List and modify custom repositories"